frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareTitleDetailsPage.tsx

```diff
@@ -139,15 +139,9 @@ const SoftwareTitleDetailsPage = ({
       isAvailableForInstall;
 
     if (showPackageCard) {
-      const packageCardData = getPackageCardInfo(title);
```

Comment:
Moved this helper function into the component as clean up so SoftwarePackageCard can have clean integration tests

Reply:
And it's also nice and clean to just pass in a software title to the card instead of all of the individual props